docs/documentation/tests/pom.xml

```diff
@@ -2,14 +2,6 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <properties>
-        <!-- this property prevents the warning "WARN FilenoUtil: Native subprocess control requires open access" when using JRuby for the unit tests on JDK17 -->
-        <surefire.system.args>
-            --add-opens=java.base/sun.nio.ch=ALL-UNNAMED
-            --add-opens java.base/java.io=ALL-UNNAMED
-        </surefire.system.args>
-    </properties>
```

Comment:
It should stay here, right? As we support JDK 17 and JDK 21. Or can you verify the warning is not present when using specific version of JDK 17?

As the properties are different than in https://github.com/keycloak/keycloak/pull/29927/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R215

Reply:
You are correct, I thought they were the same properties, but I misread. I've put this back.